provider/resources.go

```diff
@@ -217,13 +218,39 @@ func Provider() tfbridge.ProviderInfo {
 }
 
 func docEditRules(defaults []tfbridge.DocsEdit) []tfbridge.DocsEdit {
-	return append(defaults, tfbridge.DocsEdit{
-		Path: "*",
-		Edit: func(_ string, b []byte) ([]byte, error) {
-			b = bytes.ReplaceAll(b,
-				[]byte("Created using the Datadog provider in Terraform"),
-				[]byte("Created using the Datadog provider in Pulumi"))
-			return b, nil
+	return append(defaults,
+		tfbridge.DocsEdit{
+			Path: "*",
+			Edit: func(_ string, b []byte) ([]byte, error) {
+				b = bytes.ReplaceAll(b,
+					[]byte("Created using the Datadog provider in Terraform"),
+					[]byte("Created using the Datadog provider in Pulumi"))
+				return b, nil
+			},
 		},
-	})
+		removeTutorialLink,
+		removeTFOrLater,
+	)
+}
+
+var tutorialLink = "Try the hands-on tutorial on the Datadog provider on the HashiCorp Learn site."
+
+// Removes a reference to Consul Remote State Backend
+var removeTutorialLink = tfbridge.DocsEdit{
+	Path: "index.md",
+	Edit: func(_ string, content []byte) ([]byte, error) {
+		before, after, _ := bytes.Cut(content, []byte(tutorialLink))
+		content = append(before, after...)
+		return content, nil
+	},
+}
+
+// Removes a reference to TF version and compatibility
+var TFVersionOrLaterRegexp = regexp.MustCompile(`It requires [tT]erraform [0-9]+\.?[0-9]?\.?[0-9]? or later.`)
+var removeTFOrLater = tfbridge.DocsEdit{
+	Path: "index.md",
+	Edit: func(_ string, content []byte) ([]byte, error) {
+		content = TFVersionOrLaterRegexp.ReplaceAllLiteral(content, nil)
```

Comment:
This variable shouldn't be imported, so let's keep it private.
```suggestion
var tfVersionOrLaterRegexp = regexp.MustCompile(`It requires [tT]erraform [0-9]+\.?[0-9]?\.?[0-9]? or later.`)
var removeTFOrLater = tfbridge.DocsEdit{
	Path: "index.md",
	Edit: func(_ string, content []byte) ([]byte, error) {
		content = tfVersionOrLaterRegexp.ReplaceAllLiteral(content, nil)
```

Reply:
Removed as this is now in the defaults in the bridge installation-docs transformations.